src/resolvers/auth.js

```diff
@@ -36,6 +38,11 @@ const authResolvers = {
 
         addAccessTokenCookie(res, accessToken);
         addRefreshTokenCookie(res, refreshToken);
+
+        const message = createVerificationEmail(user.verificationHash);
+        const verificationEmail = { to: { email }, ...message };
+        sendEmail(verificationEmail);
+
```

Comment:
Ohhh yeah one more ask that is "nice to have" if you're able to implement it quickly: would it be possible to set emails from a certain email address domain to `verified = true` by default? i.e. from @example.com — this would just make it a ton easier when testing! If we can't do this though that's ok

Reply:
Emails with @test.com domain will be verified by default!